app/models/remote_record.rb

```diff
@@ -16,9 +16,31 @@ def self.bibdata?(source_metadata_identifier)
     PulMetadataServices::Client.bibdata?(source_metadata_identifier)
   end
 
+  def self.pulfa?(source_metadata_identifier)
+    return false if source_metadata_identifier.match?(/\//)
+    source_metadata_identifier.match?(/^([A-Z][a-zA-Z0-9\.-]+)(_[c0-9]+)?/)
```

Comment:
Are we super confident this'll work? One reason we had the fallback from what I remember was that things were kind of inconsistent in component ID land.

Reply:
I just looked at the finding aids EAD XML and there are some variations (only numbers, or starting with "i" instead of "c").  I'll update the regex to be more lenient — probably any alphanumeric string.